src/spikeinterface/core/analyzer_extension_core.py

```diff
@@ -522,6 +522,55 @@ def _select_extension_data(self, unit_ids):
 
         return new_data
 
+    def get_templates(self, unit_ids=None, operator="average"):
+        """
+        Return average templates for multiple units.
+
+        Parameters
+        ----------
+        unit_ids: list or None, default: None
+            Unit ids to retrieve waveforms for
+        operator: str
+            MUST be "average" (only one supported by fast_templates)
+            The argument exist to have the same signature as ComputeTemplates.get_templates
+
+        Returns
+        -------
+        templates: np.array
+            The returned templates (num_units, num_samples, num_channels)
+        """
+
+        assert (
+            operator == "average"
+        ), f"Analyzer extension `fast_templates` only works with 'average' templates. Given operator = {operator}"
+        templates = self.data["average"]
+
+        if unit_ids is not None:
+            unit_indices = self.sorting_analyzer.sorting.ids_to_indices(unit_ids)
+            templates = templates[unit_indices, :, :]
+
+        return np.array(templates)
+
+    def get_unit_template(self, unit_id):
```

Comment:
@DradeAW the `ComputeTemplates` doesn't have this method. Can you either add it there too or remove it?

Reply:
Done!